eclass-writing/text.xml

```diff
@@ -94,6 +108,12 @@ A simple way to verify syntax is <c>bash -n foo.eclass</c>.
 <title>Removing Eclasses</title>
 <body>
 
+<note>
+Before considering removal, please mark the eclass as <c># @DEPRECATED</c>
+to ensure consumers are warned if they are still using the eclass. Do this
+for a period before removing the eclass.
+</note>
```

Comment:
Again, this note shouldn't go to the beginning of the section.

Reply:
where should it go instead then? Please provide actual suggestions instead of just pure criticism

Reply:
See above:

> I've seen you adding notes to the very beginning of sections. IMHO that's less than optimal. At least the basic concepts of the section should be explained first (and often the note will only be understandable after reading that explanation first).
>
> So, typically notes should go after the paragraph/subsection/section they apply to.

Reply:
It would've been helpful to directly specify where in this case, as I wasn't really sure where could be more clear than the beginning.